bazel/external/wamr_llvm.BUILD

```diff
@@ -10,30 +10,27 @@ filegroup(
 )
 
 cmake(
-    name = "llvm_13_0_1_lib",
+    name = "llvm_lib",
```

Comment:
There is already `llvm_lib` defined in `llvm.BUILD` (used by WAVM), so this conflict probably breaks multi-engine builds.